app/src/main/res/layout/activity_main.xml

```diff
@@ -26,6 +26,7 @@
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:fitsSystemWindows="true"
```

Comment:
Do we need this?

Reply:
Yes otherwise toolbar doesn't layout out correctly with e2e.